features/required_tags_starts_with.feature

```diff
@@ -0,0 +1,154 @@
+Feature: Required Tags Starts With
+  As a tester I dont want to miss certain tags on my scenarios
+
+  Background: Prepare Testee
+    Given a file named ".gherkin_lint.yml" with:
+    """
+    ---
```

Comment:
this configuration file here does not test the required tags start with Feature. The configuration Thing should be tested separately.